src/common/options/mds-client.yaml.in

```diff
@@ -355,7 +355,7 @@ options:
     permissions checking, instead of relying on the permissions enforcement in
     FUSE. Set to ``false`` together with the ``client acl type=posix_acl``
     option to enable POSIX ACL.
-  default: false
+  default: true
```

Comment:
We need to keep testing our own permissions checking, and relying on the VFS for permissions checking has led to issues with coherency and liveness in the past, so we really can't turn this option off.

Reply:
Yeah, makes sense to me. We really have found several issues with this. I will remove this change since the second commit will fix this issue too.